integrationTests/relayers/slowTests/10Upgrade/tokens.go

```diff
@@ -0,0 +1,648 @@
+package upgrade
+
+import (
+	"math/big"
+
+	"github.com/multiversx/mx-bridge-eth-go/integrationTests/relayers/slowTests"
+	"github.com/multiversx/mx-bridge-eth-go/integrationTests/relayers/slowTests/framework"
+	logger "github.com/multiversx/mx-chain-logger-go"
+)
+
+var (
+	log = logger.GetOrCreate("integrationTests/relayers/slowTests")
```

Comment:
not used

Reply:
moved in the file that really needs it and has the go build condition